src/java/org/apache/cassandra/concurrent/ExecutionFailure.java

```diff
@@ -69,6 +69,10 @@ public static void handle(Throwable t)
         }
         catch (Throwable shouldNeverHappen)
         {
+            // Testing note... jvm-dtest can throw a InstanceShutdown exception from JVMStabilityInspector; this is
```

Comment:
i was told this variables name caused confusion, so adding a doc explaining things.